### ISSUE TITLE
Forces uninstall of Discord client.

### ISSUE THREAD

**Issue by PapaSmurf-ML64** [Maia-Everett/xivlauncher-core-ubuntu-packaging#1]
![image](https://user-images.githubusercontent.com/4394814/188478112-aeed1584-a3c0-4bca-b521-2fdedb640dd5.png)
Reinstalled Discord and it forcibly removes xivlauncher-core.

**Comment by Maia-Everett** [comment]
This is, sadly, not our bug, but an issue with Ubuntu's dotnet-runtime-6.0 package, on which xivlauncher-core depends. It depends on libunwind-13, which isn't compatible with libunwind-14, on which Discord depends.

https://bugs.launchpad.net/ubuntu/+source/dotnet6/+bug/1984450

One workaround would be to install dotnet-runtime-6.0 from the Microsoft repository, which doesn't depend on libunwind-13.

https://askubuntu.com/questions/1422947/why-dont-any-of-these-methods-work-for-installing-net-core-sdk-runtime-on-22/1423089#1423089

**Comment by PapaSmurf-ML64** [comment]
Thanks, that solved the problem, gotta love Ubuntu's packages breaking things. For further reference to anyone else experiencing this, from https://github.com/dotnet/core/issues/7699 :

You can install .NET 6 on Ubuntu 22.04 from packages.microsoft.com using the following steps:

```
wget https://packages.microsoft.com/config/ubuntu/22.04/packages-microsoft-prod.deb -O packages-microsoft-prod.deb
sudo dpkg -i packages-microsoft-prod.deb
rm packages-microsoft-prod.deb
```

Configure apt preferences, to prioritize PMC repository, by creating the following file: /etc/apt/preferences

To create the file:

```
sudo touch /etc/apt/preferences
```

with contents:

```
Package: *
Pin: origin "packages.microsoft.com"
Pin-Priority: 1001
```

**Comment by mirespace** [comment]
Hi all, 

ofyi, the mentioned bug (https://bugs.launchpad.net/ubuntu/+source/dotnet6/+bug/1984450) is fixed as of the latest released version of the dotnet6 package (6.0.109).
